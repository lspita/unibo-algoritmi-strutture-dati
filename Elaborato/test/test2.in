65
42
10
10
100 100 100 100 100 100 100 100 100 100
-10 -20 -20 -20 -20 -20 -20 -20 -20 100
100 100 100 100 100 100 100 100 -20 100
100 100 100 100 -20 -20 -20 100 -20 100
100 100 100 100 -20 100 100 100 -20 100
100 100 100 100 -20 100 -20 -20 -20 100
100 100 100 100 -20 100 100 100 100 100
100 -20 -20 -20 -20 -20 -20 -20 -20 -20
100 100 100 100 -20 100 100 100 100 100
100 100 100 100 -10 100 100 100 100 100