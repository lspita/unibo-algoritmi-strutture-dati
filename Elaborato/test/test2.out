0 0
0 1
0 2
0 3
0 4
0 5
0 6
0 7
0 8
0 9
1 9
2 9
3 9
4 9
5 9
6 9
6 8
6 7
6 6
6 5
5 5
4 5
4 6
4 7
3 7
2 7
2 6
2 5
2 4
2 3
3 3
4 3
5 3
6 3
6 2
6 1
6 0
7 0
8 0
9 0
9 1
9 2
9 3
9 4
9 5
9 6
9 7
9 8
9 9
-1 -1
1019585
